src/components/IncompleteSubReqCourse.vue

```diff
@@ -1 +1,187 @@
-<!-- Component for Incomplete SubReq Course -->
+<template>
+  <div class="incompletesubreqcourse">
+    <div class="draggable-requirements-wrapper"
+      v-if="subReq.displayDescription && crseInfoObjects.length > 0"
+    >
+      <div class="draggable-requirements-heading">
+        <div class="draggable-requirements-heading-label">{{ addCourseLabel }}</div>
+        <div v-if="showSeeAllLabel" class="draggable-requirements-heading-seeAll">{{ seeAll }}</div>
+      </div>
+        <div
+          v-if="display"
+          class="draggable-requirements-courses"
+          v-dragula="courseObjects"
+          bag="first-bag"
+        >
+          <div v-for="course in courseObjects" :key="course.uniqueID" class="requirements-courseWrapper">
+            <course
+              v-bind="course"
+              :courseObj="course"
+              :id="course.subject + course.number"
+              :uniqueID="course.uniqueID"
+              :compact="true"
+              :active="false"
+              class="requirements-course"
+            />
+          </div>
+        </div>
+      <div class="separator"></div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import firebase from 'firebase/app';
+// eslint-disable-next-line import/extensions
+import Course from '@/components/Course.vue';
```

Comment:
you can merge in the latest master, where I switched the linter default to always require .vue extension. Therefore, you can remove this linter error suppression line now